app/assets/javascripts/retros.js

```diff
@@ -0,0 +1,63 @@
+// Get/initialize elements
+
+document.addEventListener("DOMContentLoaded", function(event) {
+  let startNewRetroButton = document.getElementById("start-new-button");
+  let timeText = document.getElementById("time");
+
+  let timeNow;
+
+  // Set timer values
+
+  const postTimer = 1000 * 60 * 60;
+
+  // Aux functions
+
+  function checkTimeLeft() {
+    if (timeNow === postTimer) {
+      return false;
+    } else if (timeNow >= 1000 * 60 * 50) {
+      timeText.style.color = "#FF0000";
+      return true;
+    }
+  }
+
+  function getTimerValue() {
+    timeNow = timeNow + 1000;
+
+    let continueTimer = checkTimeLeft();
+
+    if (!continueTimer) {
+      clearInterval();
```

Comment:
This will not stop the interval. The `setInterval` function returns an `intervalId` which should be used to stop the interval, this is because you can have many intervals running at the same time. You can see the specs on https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/clearInterval

or you can test it by doing 

```js
const intervalId = setInterval(() => console.log("I'am running!", 2000);

clearInterval() // Will not stop the interval
clearInterval(intervalId); // Will stop the interval
```